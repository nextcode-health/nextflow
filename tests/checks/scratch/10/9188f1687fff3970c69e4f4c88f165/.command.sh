#!/bin/bash -ue
echo barcode: gamma
echo seq_ids: [two, one, three]
echo bam    : bam1 bam2 bam3
echo bai    : bai1 bai2 bai3
