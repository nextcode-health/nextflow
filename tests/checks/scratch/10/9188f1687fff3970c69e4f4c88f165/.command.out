barcode: gamma
seq_ids: [two, one, three]
bam : bam1 bam2 bam3
bai : bai1 bai2 bai3
